### PR TITLE
Add scan/insert while alter test

### PR DESCRIPTION
**Pull request by fexolm**
### Changelog entry 

...

### Changelog category 

* New feature
* Experimental feature
* Improvement
* Performance improvement
* Bugfix 
* Backward incompatible change
* Documentation (changelog entry is not required)
* Not for changelog (changelog entry is not required)

### Additional information

...

### CODE REVIEW

File: ydb/tools/olap_workload/__main__.py [hunk 1]
```diff
@@ -21,55 +22,142 @@ def __init__(self, endpoint, database, duration):
...
+    def list_tables(self):
+        db = self.driver.scheme_client.list_directory(self.database)
+        return [t.name for t in db.children if t.type == ydb.SchemeEntryType.COLUMN_TABLE]
+
+    def list_columns(self, table_name):
+        path = self.database + "/" + table_name
+        return [c.name for c in self.session.describe_table(path).columns]
+    
+    def rows_count(self, table_name):
+        return self.driver.table_client.scan_query(f"SELECT count(*) FROM {table_name}").next().result_set.rows[0][0]
```
Comment (vgvoleg) [inline ydb/tools/olap_workload/__main__.py:105]: it is better to use scan_query from session object

Comment (vgvoleg) [reply ydb/tools/olap_workload/__main__.py:105]: + for better readability you can use update query to `SELECT count(*) as count from {table_name}` to use rows[0].count

File: ydb/tools/olap_workload/__main__.py [hunk 2]
```diff
@@ -29,47 +30,138 @@ def __exit__(self, exc_type, exc_val, exc_tb):
...
+            return [c.name for c in session.describe_table(path).columns]
+
+        return self.pool.retry_operation_sync(callee)
+
+    def rows_count(self, table_name):
+        return self.driver.table_client.scan_query(f"SELECT count(*) FROM {table_name}").next().result_set.rows[0][0]
+
+    def select_10(self, table_name):
+        print(f"Select 10 {table_name}")
+        self.driver.table_client.scan_query(f"SELECT * FROM {table_name} limit 10").next()
```
Comment (zverevgeny) [inline ydb/tools/olap_workload/__main__.py:116]: Why do you specify so small limit?

File: ydb/tools/olap_workload/__main__.py [hunk 3]
```diff
@@ -29,47 +30,138 @@ def __exit__(self, exc_type, exc_val, exc_tb):
...
+
+        schema = list(schema) + ["id"]
+        column_types = ydb.BulkUpsertColumns()
+
+        for c in schema:
+            column_types.add_column(c, ydb.PrimitiveType.Int64)
+
+        data = []
+
+        for i in range(10):
```
Comment (zverevgeny) [inline ydb/tools/olap_workload/__main__.py:94]: make batch size a parameter

File: ydb/tools/olap_workload/__main__.py [hunk 4]
```diff
@@ -29,47 +30,138 @@ def __exit__(self, exc_type, exc_val, exc_tb):
...
+            data.append({c: random.randint(0, 1000000) for c in schema})
```
Comment (zverevgeny) [inline ydb/tools/olap_workload/__main__.py:95]: skip some columns

**Review by zverevgeny — DISMISSED** [review]

**Review by zverevgeny — APPROVED** [review]

**Pull request merged**